src/__mocks__/project.ts

```diff
@@ -4,6 +4,7 @@ export const mockProject = (): FullProject => ({
   secret: 'my_secret',
   enabled: true,
   requester_circleCI: null,
+  requester_gitHub: null,
```

Comment:
This is an incredible super nit, but could we make this all lowercase `requester_github`? I feel like it'd be easy to miss the "H" down the line and just want to avoid something failing because of a typo